source/scripts/components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,42 @@
+.snackbar {
+  position: fixed;
+  top: 90vh;
+  left: 50%;
+  min-width: 200px;
+  height: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #ffffff;
+  background: #323232;
+  transform: translateX(-50%);
+  animation: show-snackbar 0.5s forwards;
+}
+
+.hidden {
```

Comment:
Let's use BEM modifiers instead state classes to make clear what element we are targeting: `.snackbar_hidden`